### PR TITLE
Added Comment support via csv import

### PR DESCRIPTION
**Pull request by Fullstack-Arshad**
feature #2048

https://github.com/ghostfolio/ghostfolio/assets/61023134/73f6f168-ced2-4c1c-bfdb-39dd22ead5f9

### CODE REVIEW

**Comment by Fullstack-Arshad** [comment]
Hi @dtslvr , please let me know if any more changes is required

File: apps/client/src/app/services/import-activities.service.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ export class ImportActivitiesService {
     'unitprice',
     'value'
   ];
+  private static COMMENT_KEYS = ["comment", "note"]
```
Comment (dtslvr) [inline apps/client/src/app/services/import-activities.service.ts:31]: These values are good 👍🏻 

Can you please move this up ⬆️  to keep the alphabetical order?

File: apps/client/src/app/services/import-activities.service.ts [hunk 2]
```diff
@@ -60,7 +61,8 @@ export class ImportActivitiesService {
         symbol: this.parseSymbol({ content, index, item }),
         type: this.parseType({ content, index, item }),
         unitPrice: this.parseUnitPrice({ content, index, item }),
-        updateAccountBalance: false
+        updateAccountBalance: false,
+        comment: this.parseComment({ item })
```
Comment (dtslvr) [inline apps/client/src/app/services/import-activities.service.ts:65]: Can you please move this up ⬆️ to keep the alphabetical order?

File: apps/client/src/app/services/import-activities.service.ts [hunk 3]
```diff
@@ -128,7 +130,8 @@ export class ImportActivitiesService {
     SymbolProfile,
     type,
     unitPrice,
-    updateAccountBalance
+    updateAccountBalance,
+    comment
```
Comment (dtslvr) [inline apps/client/src/app/services/import-activities.service.ts:134]: Can you please move this up ⬆️ to keep the alphabetical order?

File: apps/client/src/app/services/import-activities.service.ts [hunk 4]
```diff
@@ -140,7 +143,8 @@ export class ImportActivitiesService {
       currency: SymbolProfile.currency,
       dataSource: SymbolProfile.dataSource,
       date: date.toString(),
-      symbol: SymbolProfile.symbol
+      symbol: SymbolProfile.symbol,
+      comment
```
Comment (dtslvr) [inline apps/client/src/app/services/import-activities.service.ts:147]: Can you please move this up ⬆️ to keep the alphabetical order?

File: apps/client/src/app/services/import-activities.service.ts [hunk 5]
```diff
@@ -197,6 +201,22 @@ export class ImportActivitiesService {
     };
   }
 
+  private parseComment({
```
Comment (dtslvr) [inline apps/client/src/app/services/import-activities.service.ts:204]: Can you please move this fuction up ⬆️ to keep the alphabetical order? (before `parseCurrency()`)

File: apps/client/src/app/services/import-activities.service.ts [hunk 6]
```diff
@@ -210,7 +230,7 @@ export class ImportActivitiesService {
   }
 
   private parseDate({
-    content,
+  content,
```
Comment (dtslvr) [inline apps/client/src/app/services/import-activities.service.ts:233]: ❓

File: test/import/ok.csv [hunk 7]
```diff
@@ -1,5 +1,5 @@
-Date,Code,Currency,Price,Quantity,Action,Fee
-16-09-2021,MSFT,USD,298.580,5,buy,19.00
-17/11/2021,MSFT,USD,0.62,5,dividend,0.00
+Date,Code,Currency,Price,Quantity,Action,Fee,Note
+16-09-2021,MSFT,USD,298.580,5,buy,19.00,Test Note 1
```
Comment (dtslvr) [inline test/import/ok.csv:2]: I would just write _My first order_ and remove the other test notes

**Review by dtslvr — CHANGES_REQUESTED** [review]
Looks already good, also thanks for adding a video! 😃

I have added some comments.

File: apps/client/src/app/services/import-activities.service.ts (hunk 1 above)
Comment (Fullstack-Arshad) [reply apps/client/src/app/services/import-activities.service.ts:31]: sure

File: apps/client/src/app/services/import-activities.service.ts (hunk 6 above)
Comment (Fullstack-Arshad) [reply apps/client/src/app/services/import-activities.service.ts]: this was accidental 😶, will cross check twice next time, for now I reverted it

**Comment by Fullstack-Arshad** [comment]
I have added all the required changes, please go through once

**Comment by dtslvr** [comment]
Regarding

> **Build code / build (18) (pull_request)** Failing after 2m

You need to replace the `"` (double quotes) with `'` (single quotes)...

**Comment by Fullstack-Arshad** [comment]
done

**Comment by Fullstack-Arshad** [comment]
Hi @dtslvr, If there is still some issue with the PR, please let me know I will fix it 🙂. I am actually new to open source project, so I don't have much idea about linting rules used in Open Source Project.

File: apps/client/src/app/services/import-activities.service.ts [hunk 8]
```diff
@@ -15,6 +15,7 @@ import { catchError } from 'rxjs/operators';
 })
 export class ImportActivitiesService {
   private static ACCOUNT_KEYS = ['account', 'accountid'];
+  private static COMMENT_KEYS = ['comment', 'note']
```
Comment (dtslvr) [inline apps/client/src/app/services/import-activities.service.ts:18]: Looks like a semicolon is missing here... 🙂

**Review by dtslvr — CHANGES_REQUESTED** [review]

File: apps/client/src/app/services/import-activities.service.ts (hunk 8 above)
Comment (Fullstack-Arshad) [reply apps/client/src/app/services/import-activities.service.ts]: ok, This time I have used prettier to fix all the linting issue related to my code. I hope things will be fine now... 🤞

**Review by dtslvr — APPROVED** [review]
Great contribution @Fullstack-Arshad! 🙌🏻

**Pull request merged**